text/204-golang-bindings.md

```diff
@@ -1159,8 +1394,6 @@ also true, an output value, IE a method's return value, cannot be changed from n
 With these downsides, it still seems like this is the most idiomatic route to take. Go users are familiar with using pointer types in this way from other
 popular modules. Additionally, it is the only strategy we can identify that allows us to maintain JSII's versioning compatibility strategy.
 
-### Explored Alternatives
-
 1. Don't use pointers.
```

Comment:
I read this as now being a continuation of the "explored alternatives" list, therefore this should be 2 and "wrapper types" should be 3.